content/post/67-crdt-research-meetup.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2019-01-28
+url: 67-crdt-research-meetup
+tags: crdt research meetup videos ipfs
+title: 2018 CRDT Research Meetup - Lisbon
+author: Pedro Teixeira
+---
+
+On April last year some members of the IPFS team gathered in Lisbon for a week of fun hack. During this week we mostly hacked and collaborated on some aspects of IPFS, libp2p and applications built on top of them, but also took one day to host a CRDT Research meetup.
+
+CRDT stand for Conflict-free Replicated Data Types and are a mathematical framework and building block that allows different replicas to cooperate without requiring coordination, making them very useful for creating eventually consistent data layers over intermittent and unreliable networks. For instance, CRDTs make sure that, no matter the order of the messages exchanged, all the peers eventually converge to the same state.
```

Comment:
Link `CRDT stand for Conflict-free Replicated Data Types` to Wikipedia.

Reply:
Done